Jenkinsfile

```diff
@@ -385,12 +387,18 @@ def nodeWithCleanup(String label, Closure steps) {
   }
 }
 
-def xbuildSafe(String command) {
+def xbuild(String arguments) {
   try {
-    sh "${command}"
+    sh "${xbuildCmd} ${arguments} 1> xbuildOutput"
```

Comment:
I see. You could also do something like:
```groovy
def errCode = sh returnStatus: true, script: "${xbuildCmd} ${arguments} > stdout"
def out = readFile('stdout')
```
and fail if `errCode != 0 && !out.contains(blabla)`